proxy/Dockerfile

```
@@ -0,0 +1,33 @@
+FROM rust:latest AS builder
```

Hadolint

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)